src/legacy/core_plugins/metrics/server/lib/search_strategies/searh_requests/search_request.js

```diff
@@ -22,11 +22,11 @@ import { MultiSearchRequest } from './multi_search_request';
 import { SingleSearchRequest } from './single_search_request';
 
 export class SearchRequest extends AbstractSearchRequest {
-  getSearchRequestType(options) {
-    const isMultiSearch = Array.isArray(options.body);
+  getSearchRequestType(searches) {
+    const isMultiSearch = searches && searches.length > 1;
```

Comment:
IMHO I will also keep the `Array.isArray` check because searches can also be an object with a length property `{ length: 2}`

Reply:
Thanks